test/api.mocha.coffee

```diff
@@ -350,6 +356,11 @@ describe "API", ->
 
               # Send them invitations
               (_party, cb) ->
+                 # @TODO - _party should be array of newly
```

Comment:
This explains why A) my local install suddenly had 20+ users with admin capabilities and B) why the dread dragon kept appearing in the tavern.